### PR TITLE
Fix #151: Show y-value in tooltip

### LINKED ISSUE #151: Show the y-value in the hover description at the bottom of the page
This is what the hover description looks like now:

screen shot 2017-06-29 at 10 05 15 am

We should add the y-value of the data point to that hover description. Not the percent, but the actual y-value of that data point. For example, **351 users**. I think we already have the ability to do this, but we're not doing it in the default format string.

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
I want to be sure that I understood this requirement correctly, so maybe both Robs can take a look.

I used the `p * numObs* calculation as suggested in this issue and called this value *clients* in the tooltip. Does all of that sound about right?

File: viewer/static/js/app/reducers/metric-metadata-reducer.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const getHoverStrings = (metadata) => {
   for (const id in metadata) {
     if (metadata.hasOwnProperty(id)) {
       var tooltip = metadata[id].tooltip;
-      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x} {xunit}, y={y}%, n={n}';
+      strings['id-' + id] = tooltip ? tooltip : '{pop}: x={x}{xunit}; y={y}%; clients={clients}; n={n}';
```
Comment (robhudson) [inline viewer/static/js/app/reducers/metric-metadata-reducer.js]: This is actually the number of pings that have a non-null value for this particular metric and cohort.

Comment (openjck) [reply viewer/static/js/app/reducers/metric-metadata-reducer.js]: Hmm. Is there another term that comes to mind that could be used here?

Comment (robhudson) [reply viewer/static/js/app/reducers/metric-metadata-reducer.js]: This is actually going to be related to #274 after #273 lands in the API, I think. In the last EV meeting it was mentioned that you cannot simply multiple the percentage by the total pings.

**Comment by openjck** [comment]
Experiments Viewer has been replaced by Test Tube and now redirects to it.

**Pull request closed without merging**